packages/sw-cli/src/cli/cli-help.txt

```diff
@@ -0,0 +1,14 @@
+sw-cli
```

Comment:
Not a reflection on your code at all, but it's pretty disappointing that there isn't a better solution for `node` CLI help than an external text file. I was taking a look at what `yeoman` does and I guess it uses the same approach: https://github.com/yeoman/yo/blob/master/lib/usage.txt

😦